console/src/console.rs

```diff
@@ -0,0 +1,55 @@
+use nu_ansi_term::Color::{Cyan, Red};
+
+/// Console print to users console
+pub trait Console {
+    fn print(&self);
+}
+
+/// Info prints information for users
```

Comment:
Seems a bit clunky to have to specify a struct in order in order to print these.
Why not simply expose different methods, such as the usual log crate, eg: info, warn, error?
You can also multiple args to the fn if necessary, eg: your data and message.

Reply:
modified